Permanent/Assets/Localization/en.lproj/Localizable.strings

```diff
@@ -140,6 +140,16 @@
 "RequestApproval" = "Request Approval";
 "RemoveMember" = "Remove member?";
 "Pending" = "Pending";
+"InvalidCsrf" = "Invalid CSRF";
+"IncorectOldPassword" = "The old password you have entered is incorrect";
+"LowPasswordComplexity" = "The new password does not meet complexity requirements";
```

Comment:
Are these the same messages used in the web app?

Reply:
Changed current texts with the texts from web app.